website/docs/getting-started/setup/instance-configuration/email/README.md

```diff
@@ -45,6 +45,10 @@ Be sure to double-check your configuration if you find that you're able to send
 
 ## Configure using admin settings
 
+:::info
+If you have set values using [environment variables](#environment-variables) for your instance, those values take precedence over values specified in the Admin Settings UI.
```

Comment:
this link in this callout is not working. Just change the headings in the page to match the other pages and should solve the issue